src/ConfigUtil.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Drupal\oe_bootstrap_theme;
+
+use Drupal\Core\Config\FileStorage;
+use Drupal\Component\Utility\Crypt;
+
+/**
+ * Utility class to import configuration from file.
+ */
+class ConfigUtil {
```

Comment:
I've already reviewed this file, could you drop me a link here so I dont go into it again pls

Reply:
We solved stuff at 
* https://github.com/openeuropa/oe_bootstrap_theme/pull/123 (OEL-759, Facts and figures)